packages/yoga/src/Datepicker/web/Calendar.jsx

```diff
@@ -205,19 +205,26 @@ function Calendar({
     return new Date(Date.UTC(year, month, day)).getTime();
   };
 
+  const isSameDate = (date1, date2) =>
+    date1.getDate() === date2.getDate() &&
+    date1.getMonth() === date2.getMonth() &&
+    date1.getFullYear() === date2.getFullYear();
```

Comment:
This function doesn't have to be declared inside render.

Reply:
Done